### ISSUE TITLE
Glyphicons font files missing but generated oscar/css/styles.css still references them

### ISSUE THREAD

**Issue by davidmarquis** [django-oscar/django-oscar#1802]
The following 4 files are referenced in the generated `oscar/css/styles.css' file but aren't packaged with Oscar:

- glyphicons-halflings-regular.eot
- glyphicons-halflings-regular.svg
- glyphicons-halflings-regular.ttf
- glyphicons-halflings-regular.woff

I'm using a CSS post-processor (django-pipeline) and it complains that these files are missing while inspecting the CSS during post-processing for changing references to static files for versioned files (files with a hash in the name as a cache buster mechanism). This effectively crashes the `collectstatic` utility and prevents the build. I had to download these static files from the [bootstrap3](https://github.com/twbs/bootstrap/tree/master/fonts) project and include it in my project's static directory under `oscar/fonts/...` for the `collectstatic` utility to complete.

The stacktrace is as follows:

```
Post-processing 'oscar/css/styles.css' failed!

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.4/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/usr/local/lib/python3.4/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.4/site-packages/django/core/management/base.py", line 533, in handle
    return self.handle_noargs(**options)
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 168, in handle_noargs
    collected = self.collect()
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 120, in collect
    raise processed
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/storage.py", line 242, in post_process
    content = pattern.sub(converter, content)
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/storage.py", line 181, in converter
    hashed_url = self.url(unquote(joined_result), force=True)
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/storage.py", line 128, in url
    hashed_name = self.stored_name(clean_name)
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/storage.py", line 277, in stored_name
    cache_name = self.clean_name(self.hashed_name(name))
  File "/usr/local/lib/python3.4/site-packages/django/contrib/staticfiles/storage.py", line 91, in hashed_name
    (clean_name, self))
ValueError: The file 'oscar/fonts/glyphicons-halflings-regular.woff' could not be found with <pipeline.storage.PipelineCachedStorage object at 0x7f534c6068d0>.
```

**Comment by maikhoepfel** [comment]
Agreed, I just ran into this. DRF had the same problem: https://github.com/tomchristie/django-rest-framework/issues/2008

**Comment by maikhoepfel** [comment]
I just released [Oscar 1.1.1](https://django-oscar.readthedocs.org/en/releases-1.1/releases/v1.1.1.html) to fix your issue. Thanks for the good issue report!

**Comment by davidmarquis** [comment]
Will try that out soon! Thanks for the good fix :)
